src/components/Scorecards/ScorecardCard/ScorecardCard.tsx

```diff
@@ -28,9 +28,10 @@ export const ScorecardCard = ({ scorecard }: ScorecardCardProps) => {
 
   return (
     <ListCard
-      name={scorecard.name}
       creatorName={scorecard.creator.name}
       description={scorecard.description}
+      name={scorecard.name}
+      truncateToCharacters={200}
```

Comment:
This limit is somewhat arbitrary (it's based on what we use in the Cortex web app) -- let me know if you have any suggestions!